src/client/components/Forms/SignUp.js

```diff
@@ -1,54 +1,175 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import iconGithub from '../../assets/images/icons/github.svg';
+import './SignUp.style.css';
+import Button from '../Button/Button';
 
 export default function SignUp({ onSubmit }) {
+  const [name, setName] = useState('');
   const [email, setEmail] = useState('');
+  const [position, setPosition] = useState('');
+  const [image, setImage] = useState('');
+  const [linkedin, setLinkedin] = useState('');
+  const [github, setGithub] = useState('');
   const [password, setPassword] = useState('');
   const [passwordConfirm, setPasswordConfirm] = useState('');
+  const [validation, setValidation] = useState('');
+
   const handleEmailInput = (e) => setEmail(e.target.value);
+  const handleNameInput = (e) => setName(e.target.value);
+  const handlePositionInput = (e) => setPosition(e.target.value);
+  const handleImageInput = (e) => setImage(e.target.value);
+  const handleLinkedinInput = (e) => setLinkedin(e.target.value);
+  const handleGithubInput = (e) => setGithub(e.target.value);
   const handlePasswordInput = (e) => setPassword(e.target.value);
   const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
   const handleSubmit = (e) => {
     e.preventDefault();
+    if (password !== passwordConfirm) {
+      return setValidation('**Your password does not match**');
+    }
+    setValidation('');
     onSubmit({ email, password, passwordConfirm });
```

Comment:
I would like to suggest that onSubmit must be called [with all these parameters](https://github.com/HackYourFuture-CPH/rate-my-cv/blob/bc6da77062d761c30605674bc6d5c6bd6a8dc42e/src/client/containers/SignUp/index.js#L13)

Reply:
I am not sure about this I am just use  validation in password

Reply:
im not sure what do you mean?

Reply:
hope now it makes sense? Please have a look at [my PR again](https://github.com/HackYourFuture-CPH/rate-my-cv/blob/bc6da77062d761c30605674bc6d5c6bd6a8dc42e/src/client/containers/SignUp/index.js#L13)